02-peer-protocol.md

```diff
@@ -2123,30 +2344,49 @@ leaking information to senders trying to probe the blinded route.
 
 When a node has changes for the remote commitment, it can apply them,
 sign the resulting transaction (as defined in [BOLT #3](03-transactions.md)), and send a
-`commitment_signed` message.
+`commitment_signed` message bundle. First it will send `commitment_signed`
+for the active channel, then it will send another `commitment_signed` message
+for each splice candidate awaiting confirmation.
 
 1. type: 132 (`commitment_signed`)
 2. data:
    * [`channel_id`:`channel_id`]
    * [`signature`:`signature`]
    * [`u16`:`num_htlcs`]
    * [`num_htlcs*signature`:`htlc_signature`]
+   * [`commitment_signed_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `commitment_signed_tlvs`
+2. types:
+    1. type: 0 (`splice_info`)
+    2. data:
+        * [`channel_id`:`splice_channel_id`]
```

Comment:
As previously discussed, I think we should rather use a `batch_size` here, because otherwise you cannot properly deal with concurrency issues between `splice_locked` and `commit_sig` and may deadlock.

We need to specify in which order `commit_sig`s are sent in that "batch": eclair currently sends them ordered by sending the `commit_sig` matching the latest splice first, and the older ones afterwards.